website/static/js/pages/prereg-admin-page.js

```diff
@@ -38,10 +41,9 @@ function adminView(data) {
 
     // variables for editing items in row
     self.edit = ko.observable(false);
-    self.item = ko.observable();
     self.commentsSent = ko.observable('no');
```

Comment:
For you it may be easier to just extend the Draft model (just like how we created Uploader as an extension of Question)